<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateGaugeDialog</class>
 <widget class="QDialog" name="CreateGaugeDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Gauge</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" rowspan="2">
    <widget class="QStackedWidget" name="stackedWidget"/>
   </item>
   <item row="0" column="0">
    <widget class="QListWidget" name="gaugeList">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>200</height>
      </size>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="currentRow">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>Radial</string>
      </property>
      <property name="icon">
       <iconset resource="../zenom.qrc">
        <normaloff>:/gauge/resources/radialgauge.png</normaloff>:/gauge/resources/radialgauge.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Linear</string>
      </property>
      <property name="icon">
       <iconset resource="../zenom.qrc">
        <normaloff>:/gauge/resources/lineargauge.png</normaloff>:/gauge/resources/lineargauge.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Numeric</string>
      </property>
      <property name="icon">
       <iconset resource="../zenom.qrc">
        <normaloff>:/gauge/resources/numericgauge.png</normaloff>:/gauge/resources/numericgauge.png</iconset>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="LogVariableTreeWidget" name="logVariableList">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <column>
      <property name="text">
       <string notr="true">Log Variables</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LogVariableTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>widget/logvariabletreewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../zenom.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CreateGaugeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CreateGaugeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
